src/domain/session/room/timeline/TimelineViewModel.js

```diff
@@ -47,6 +47,58 @@ export class TimelineViewModel extends ViewModel {
         this._requestedEndTile = null;
         this._requestScheduled = false;
         this._showJumpDown = false;
+        this._watchMap = new WeakMap();
```

Comment:
My understanding is that if we use an object as key here, it should be a weak reference and therefore shouldn't prevent the GC from releasing that object. But logging the contents of `this._watchMap` after the gap has filled ( and tile disposed ) still shows the object there? I did manually trigger garbage collection as well.